### PR TITLE
migration to elasticsearch 7.5

### PR DESCRIPTION
**Pull request by elie29**
Hello Stephane,

This code works correctly on bonsai elasticsearch 7.5. Starting from 7+, the existing code won't work as bonsai has merged their clusters to 7.5

Regards,
Elie.

### CODE REVIEW

File: kafka-consumer-elasticsearch/src/main/java/com.github.simplesteph.kafka/tutorial3/ElasticSearchConsumer.java [hunk 1]
```diff
@@ -129,11 +129,9 @@ public static void main(String[] args) throws IOException {
                     String id = extractIdFromTweet(record.value());
 
                     // where we insert data into ElasticSearch
-                    IndexRequest indexRequest = new IndexRequest(
-                            "twitter",
-                            "tweets",
-                            id // this is to make our consumer idempotent
-                    ).source(record.value(), XContentType.JSON);
+                    IndexRequest indexRequest = new IndexRequest("twitter")
```
Comment (simplesteph) [inline kafka-consumer-elasticsearch/src/main/java/com.github.simplesteph.kafka/tutorial3/ElasticSearchConsumer.java]: is this API also in the old clients? or is this something new?
Where's the `tweets` parameter going?

Comment (elie29) [reply kafka-consumer-elasticsearch/src/main/java/com.github.simplesteph.kafka/tutorial3/ElasticSearchConsumer.java]: Well the new API, we cannot add a type and we need to make an index for each type we used before. So if we had /twitter/tweets and /twitter/users per example we need to split them and create two indexes. More explanation here: https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html

Comment (simplesteph) [reply kafka-consumer-elasticsearch/src/main/java/com.github.simplesteph.kafka/tutorial3/ElasticSearchConsumer.java]: Thanks for explaining. As long as you can comment the old code and add comprehensive comments for the changes , I'm happy to take this PR

Comment (elie29) [reply kafka-consumer-elasticsearch/src/main/java/com.github.simplesteph.kafka/tutorial3/ElasticSearchConsumer.java]: ok @simplesteph I will do this and update the PR

**Comment by simplesteph** [comment]
Thank you!

**Pull request merged**